tests/hardhat/Tokens/borrowAndRepayTest.ts

```diff
@@ -298,9 +405,94 @@ describe("VToken", function () {
     });
   });
 
+  describe("repayBorrowFresh for stable rate borrowing", () => {
```

Comment:
I think it would make sense to add some tests checking specifically the accrued stable interests along the blocks, including simulations with different borrows with different stable rates. What do you think?

Reply:
Yes, I have added some tests [here](https://github.com/VenusProtocol/isolated-pools/pull/132/commits/ac0e73aaef283a22f39acfda1d5cff5572b65ea4).